war/src/main/less/base/layout-commons.less

```diff
@@ -76,7 +76,7 @@ body {
 }
 
 #main-panel {
-  padding: 15px 15px 40px 15px;
+  padding: 30px;
```

Comment:
might as well use `2rem` here

Reply:
Also, I'm a bit -1 on this change. This will affect everything, including page footers, etc.

Reply:
I've changed it to 2rem. The padding will affect every page, but I think it's worth it as it improves the spacing of Jenkins' UI and gives it more balance.